src/JsonLD.Entities/Converters/JsonLdLiteralConverter.cs

```diff
@@ -28,7 +29,28 @@ static JsonLdLiteralConverter()
         /// </summary>
         public override void WriteJson(JsonWriter writer, [AllowNull] object value, JsonSerializer serializer)
         {
-            writer.WriteValue(value);
+            var type = value?.GetType();
+            if (type?.HasImplicitlyTypedJsonType() == true)
+            {
+                writer.WriteValue(value);
+                return;
+            }
+
+            writer.WriteStartObject();
+            writer.WritePropertyName(JsonLdKeywords.Value);
+
+            if (value is TimeSpan)
+            {
+                writer.WriteValue(XmlConvert.ToString((TimeSpan)value));
+            }
+            else
+            {
+                var valueString = JsonConvert.ToString(value);
+                writer.WriteValue(valueString.Trim('"')); // for the weirdest of reasons, DateTime is double-quoted
```

Comment:
Hm, that's weird. We do quite an extensive customisation of the JSON serialisation in Pomona, please have a look at that if you dare. 😄  I can give it a whack when time allows, which I won't have for a few weeks. 😕